src/utils/ScreenshotStrategyManager.js

```diff
@@ -26,8 +26,8 @@ export default class ScreenshotStrategyManager {
       return new FullpageScreenshotStrategy(screenDimensions);
     }
   
-    const { isIOS } = browser;
-    if (isIOS) {
+    const { isMobile } = browser;
+    if (isMobile) {
```

Comment:
Is this really necessary for android simulators?
For real android devices with chrome this shouldn't be executed, merging ist enough.